src/NzbDrone.Core/ImportLists/FetchAndParseImportListService.cs

```diff
@@ -49,10 +49,15 @@ public List<ImportListItemInfo> Fetch()
                 var importListLocal = importList;
                 var importListStatus = _importListStatusService.GetLastSyncListInfo(importListLocal.Definition.Id);
 
-                if (importListStatus.HasValue && DateTime.UtcNow < (importListStatus + importListLocal.MinRefreshInterval))
+                if (importListStatus.HasValue)
                 {
-                    _logger.Trace("Skipping refresh of Import List {0} due to minimum refresh inverval", importListLocal.Definition.Name);
-                    continue;
+                    var importListNextSync = importListStatus.Value + importListLocal.MinRefreshInterval;
+
+                    if (DateTime.UtcNow < importListNextSync)
+                    {
+                        _logger.Trace("Skipping refresh of Import List [{0}]: {1} due to minimum refresh interval. Next Sync at {2}", importList.Name, importListLocal.Definition.Name, importListNextSync);
```

Comment:
The next sync is unlikely to be at exactly that time and should reflect a fuzzier time. `Sync` also doesn't need to be capitalized.